### DISCUSSION TITLE
Teleport Connect early feedback

### DISCUSSION THREAD

**Discussion by klizhentas** [gravitational/teleport#12619] [General]
If you have tried [Teleport Connect](https://goteleport.com/connect/), or want to try it, please post your feedback here!

Right now it only works on MacOS, but we will be working on Windows and Linux builds soon.

**UPDATE (October 4th, 2022)**

Teleport connect works on Windows and Linux, check out [our downloads page](https://goteleport.com/download/)

**Comment by Sakuya** [comment]
Hello.
Thank you very much. 

Please add support for custom fonts.
![image](https://user-images.githubusercontent.com/631302/191071913-cc4e3eba-92a9-4b7d-8af2-e98e0c6552ea.png)

**Reply by zmb3** [reply]
Hi @Sakuya - we recently added support for customizing the terminal font. You can find more info on this (and other) configuration in our docs: https://goteleport.com/docs/connect-your-client/teleport-connect/#configuration
